### ISSUE TITLE
SSH/SDK Mode only working after flash

### ISSUE THREAD

**Issue by eLtMosen** [AsteroidOS/meta-sparrow-hybris#11]
Using the nightly from 2021-12-24 on sparrow.
SDK Mode connection via ssh works right after the flash but fails to connect after reboot.

A: Observations during malfunctioning sdk mode:
- Cable is having good connection, checked using `fastboot devices` multiple times
- There is no "Using SDK Mode" message when plugin in the usb cable
- Spaming `ssh ceres@192.168.2.15` during boot and for a minute after boot does always result in `ssh: connect to host 192.168.2.15 port 22: Network is unreachable`.
- Plug usb cable in and out does not help
- Multiple reboots did not help
- Booting with cable not plugged in, plugging it in some seconds after boot does not help.
- Toggling usb mode to "charge only" and back does not help

B: Even after reflashing to fresh state, sometimes:
- right after first boot, with the usb cable still connected from fastboot flash, the SDK Network shows up on my linux computer
- But there is also no connection possible `ssh: connect to host 192.168.2.15 port 22: No route to host`
- Plugging the cable in and out again results in `Network is unreachable` with no network showing up on computer.

### I can reliably get SDK mode to work on first boot after flash now by disconnecting the usb cable after flash and reconnect it after it settled after boot. 
C: During runtime with working SDK Mode:

- It persists ceres restart
- But when plugging the cable in and out, there is no further connection possible `Network is unreachable`
- Then rebooting from settings and booting with usb not connected and letting it settle, sadly does not have the same success but shows behaviour described in A:

**Comment by eLtMosen** [comment]
tl:dr; Is this a USB2 (not working) vs USB3 (working) issue?

### Log from 1.0 fastboot-image and 2021-12-12-24 user-image
- booting the watch with
- Letting the device settle for 30sec
- plugging in USB and wait for PC to notify about SDK Network being available
- connecting to `ssh ceres@192.168.2.15`, going `su` and executing `journalctl --no-pager > journal-fastboot1.log`
[journal-fastboot1.log](https://github.com/AsteroidOS/meta-sparrow-hybris/files/7780404/journal-fastboot1.log)

### Log from both nightly images flashed:
- Initial boot after flash with usb still plugged in from flash
- skip tutorial
- Try to connect via ssh but result is ```ssh ceres@192.168.2.15
ssh: connect to host 192.168.2.15 port 22: Network is unreachable```
- Remove usb cable and plug it back in (14:56)
- successfull connect via ssh and going `su`
- doing `journalctl --no-pager > journal-fastboot2.log`
[journal-fastboot2.log](https://github.com/AsteroidOS/meta-sparrow-hybris/files/7780467/journal-fastboot2.log)

- Then removing and plugging back the USB does not work and i get `Network is unreachable` even after waiting 90sec.
- Rebooting with usb cable attached during boot results in not working SDK-Mode after boot
- Plugging USB out and in  -> then brings back SDK Mode to working state (~15:03).
[journal-fastboot3.log](https://github.com/AsteroidOS/meta-sparrow-hybris/files/7780487/journal-fastboot3.log)

### New realisation, the SDK-Mode start up depends on wether i choose USB2 or USB3 port:

- **All above depended on using a powered USB2 port on my laptop**.
- When switching to USB3 port all works as expected.
Log from several failed attempts using USB2 and successfull ssh and journalctl-log creation using USB3 connection:
[journal-fastboot4.log](https://github.com/AsteroidOS/meta-sparrow-hybris/files/7780544/journal-fastboot4.log)
